lib/components/inputs/TagsBox/TagsBox.tsx

```diff
@@ -137,13 +145,14 @@ function TagsBox(props: TagsBoxProps) {
           <div onClick={onMouseClick} className='tags-user-input-preview'>
             {`'${userInput}'`}
             {editValueError && (
-              <span className='tags-invalid'>{` - ${
-                invalidTagText || 'Invalid value'
-              }`}</span>
+              <span className='tags-invalid'>
+                {` - ${invalidTagText || 'Invalid value'}`}
+              </span>
             )}
           </div>
         )}
         classNamePrefix='react-tagsbox'
+        menuPortalTarget={document.body}
```

Comment:
also here - what does this influence?

Reply:
It should place the portal inside the current body, other wise it is not visible 
![image](https://github.com/user-attachments/assets/3f2b9f00-f637-4c87-8ddb-37d28deb0ad5)

Reply:
seems visible...also, the dropdown should look like this:
<img width="401" alt="Screenshot 2024-09-10 at 14 11 45" src="https://github.com/user-attachments/assets/e1edcc61-be59-4430-8c81-e2b82dc5a274">

Reply:
In that case it is pushing the entire document down, and not using portal, I'll keep it as an optional value